html/index.html

```diff
@@ -110,24 +105,10 @@ <h4>Contact</h4>
         </div>
     </header>
 
-    <main>
-
-        <section class="py-5 text-center container">
-            <div class="row py-lg-5">
-                <div class="col-lg-6 col-md-8 mx-auto">
-                    <h1 class="fw-light">Album example</h1>
-                    <p class="lead text-body-secondary">Something short and leading about the collection below—its
-                        contents, the creator, etc. Make it short and sweet, but not too short so folks don’t simply
-                        skip over it entirely.</p>
-                    <p>
-                        <a href="#" class="btn btn-primary my-2" id="main-call-button">Main call to action</a>
-                        <a href="#" class="btn btn-secondary my-2">Secondary action</a>
-                    </p>
-                </div>
-            </div>
-        </section>
+    <main> 
 
-        <div class="album py-5 bg-body-tertiary">
+        <div class="album py-5 bg-body-tertiary">      
+          <h1> List of Videos </h1>
```

Comment:
I've looked at the solution it looks great. However there is a small styling issue with this h1 as it should be on the same level as the first video:
<img width="1450" alt="Screenshot 2023-10-06 at 18 10 16" src="https://github.com/kubestaff/golearn/assets/144615047/abc5bb2d-24a3-4e6b-97ae-d6bdf65a6fae">